### PR TITLE
Add Get-FGTUser[LDAP|Groups|RADIUS] and Get-FGTSystemAdmin

### PR DESCRIPTION
**Pull request by Cool34000**
Retrieve LDAP servers
Retrieve RADIUS servers
Retrieve User groups

### CODE REVIEW

File: PowerFGT/Public/cmdb/user/radius.ps1 [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            $invokeParams.add( 'vdom', $vdom )
+        }
+
+        #Filtering
+        switch ( $PSCmdlet.ParameterSetName ) {
+            "name" {
+                $filter_value = $name
+                $filter_attribute = "name"
+            }
+            "id" {
```
Comment (alagoutte) [inline PowerFGT/Public/cmdb/user/radius.ps1:79]: same here

**Review by alagoutte — CHANGES_REQUESTED** [review]

File: PowerFGT/Public/cmdb/user/ldap.ps1 [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            $invokeParams.add( 'vdom', $vdom )
+        }
+
+        #Filtering
+        switch ( $PSCmdlet.ParameterSetName ) {
+            "name" {
+                $filter_value = $name
+                $filter_attribute = "name"
+            }
+            "id" {
```
[the remark this answers is not in the document]
Comment (Cool34000) [reply PowerFGT/Public/cmdb/user/ldap.ps1:79]: There is no "id" returned
I delete the first part and forgot the second part!

File: PowerFGT/Public/cmdb/user/radius.ps1 (hunk 1 above)
Comment (Cool34000) [reply PowerFGT/Public/cmdb/user/radius.ps1]: Fixed both

**Comment by alagoutte** [comment]
Thanks Sylvain, 

there is no id value ? for this call ? (i don't have yet check)

**Comment by Cool34000** [comment]
Affirmative!

```
(Invoke-FGTRestMethod -uri 'api/v2/cmdb/user/ldap').results

name                    : XXXXXX
q_origin_key            : XXXXXX
server                  : 172.16.1.1
secondary-server        :
tertiary-server         :
server-identity-check   : disable
source-ip               : 0.0.0.0
cnid                    : sAMAccountName
dn                      : DC=domain,DC=local
type                    : regular
username                : ldap@domain.local
password                : ENC XXXX
group-member-check      : user-attr
group-search-base       :
group-object-filter     : (&(objectcategory=group)(member=*))
group-filter            :
secure                  : disable
ssl-min-proto-version   : default
ca-cert                 :
port                    : 389
password-expiry-warning : disable
password-renewal        : disable
member-attr             : memberOf
account-key-processing  : same
account-key-filter      : (&(userPrincipalName=%s)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))

```
```

(Invoke-FGTRestMethod -uri 'api/v2/cmdb/user/radius').results

name                          : NPS XXXXX
q_origin_key                  : NPS XXXXX
server                        : 172.16.1.1
secret                        : ENC XXXX
secondary-server              :
secondary-secret              :
tertiary-server               :
tertiary-secret               :
timeout                       : 5
all-usergroup                 : disable
use-management-vdom           : disable
nas-ip                        : 0.0.0.0
acct-interim-interval         : 0
radius-coa                    : disable
radius-port                   : 0
h3c-compatibility             : disable
auth-type                     : ms_chap_v2
source-ip                     :
username-case-sensitive       : disable
class                         : {}
password-renewal              : disable
password-encoding             : auto
acct-all-servers              : disable
rsso                          : disable
rsso-radius-server-port       : 1813
rsso-radius-response          : disable
rsso-validate-request-secret  : disable
rsso-secret                   :
rsso-endpoint-attribute       : Calling-Station-Id
rsso-endpoint-block-attribute :
sso-attribute                 : Class
sso-attribute-key             :
sso-attribute-value-override  : enable
rsso-context-timeout          : 28800
rsso-log-period               : 0
rsso-log-flags                : protocol-error profile-missing accounting-stop-missed accounting-event endpoint-block radiusd-other
rsso-flush-ip-session         : disable
rsso-ep-one-ip-only           : disable
accounting-server             : {}

```

**Pull request merged**